Sample_v3/Extensions/Channel+MembersInfo.swift

```diff
@@ -4,12 +4,25 @@
 
 import StreamChatClient
 
+/// Creates formatted string for channel members and online members.
+///
+/// Example result: ` "10 members, 3 online"`
 func createMemberInfoString(for channel: ChatChannel) -> String {
     "\(channel.members.count) members, \(channel.members.filter(\.isOnline).count) online"
 }
 
+/// Creates formatted string for currently typing members from `currentlyTypingMembers` property of `ChatChannel` if any.
+///
+/// Example result: ` "Nick is typing..."` or`"Nick, Maria are typing..."`
 func createTypingMemberString(for channel: ChatChannel) -> String? {
-    guard !channel.currentlyTypingMembers.isEmpty else { return nil }
-    let names = channel.currentlyTypingMembers.map { $0.name ?? $0.id }.sorted()
-    return names.joined(separator: ",") + " \(names.count == 1 ? "is" : "are") typing..."
+    createTypingMemberString(for: channel.currentlyTypingMembers)
+}
+
+/// Creates formatted string for currently typing members from ` Set<ChatChannelMember>` if any..
+///
+/// Example result: ` "Nick is typing..."` or `"Nick, Maria are typing..."`
+func createTypingMemberString(for members: Set<ChatChannelMember>) -> String? {
```

Comment:
Sorry, now when I see it, why we have two different helper functions doing the same thing? @mantergo

Reply:
Please pick one, it's completely unnecessary

Reply:
Fixed 👌